CHANGELOG.md

```diff
@@ -1,6 +1,9 @@
 # Change Log
 This project adheres to [Semantic Versioning](http://semver.org/).
 
+## 0.2.2
```

Comment:
Please remove ChangeLog changes. It should `0.3`.

Reply:
Changed to 0.3